sdk/tools/pefixup.c

```diff
@@ -144,9 +144,15 @@ static int driver_fixup(int mode, unsigned char *buffer, PIMAGE_NT_HEADERS nt_he
         if (Section->Characteristics & IMAGE_SCN_CNT_CODE)
             Section->Characteristics &= ~IMAGE_SCN_CNT_INITIALIZED_DATA;
 
-        /* For some reason, .rsrc is made writable by windres */
         if (strncasecmp((char*)Section->Name, ".rsrc", 5) == 0)
         {
+            /* .rsrc is discardable for driver images, WDM drivers, and Kernel-Mode DLLs */
+            if (mode == MODE_KERNELDRIVER || mode == MODE_WDMDRIVER || mode == MODE_KERNELDLL)
+            {
+                Section->Characteristics |= IMAGE_SCN_MEM_DISCARDABLE;
+            }
+
+            /* For some reason, .rsrc is made writable by windres */
```

Comment:
Why does this need an extra handling there?

Reply:
I follow the loop logic "continue if the given section can be paged".

Reply:
My comment was misplaced and was for the handling of the discardable flag.

Reply:
MSVC linker does that by default.

Reply:
Oh sorry you were fixing also another problem in the same PR.
Ok then keep it, and adjust the comment there too: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/cmake/gcc.cmake;hb=2587d72ef7c0673a619a7ad4202a82f926e8a6ab#l300